.travis.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+sudo: required
+
+language: minimal
+
+services:
+  - docker
+
+before_install:
+  # Set some vars
+  - export container_id='test-role-container'
```

Comment:
Nit: Try to use only one type of quotes unless there's a specific reason like string interpolation.